src/cts/src/TritonCTS.cpp

```diff
@@ -1179,6 +1179,14 @@ Clock TritonCTS::forkRegisterClockNetwork(
   secondNet = odb::dbNet::create(block_, newClockName.c_str());
   secondNet->setSigType(odb::dbSigType::CLOCK);
 
+  odb::dbModule* first_net_module = network_->getParentModule(firstNet);
+  odb::dbModule* second_net_module = network_->getParentModule(secondNet);
+  odb::dbModule* target_module = nullptr;
+  if ((first_net_module != nullptr)
+      && (first_net_module == second_net_module)) {
+    target_module = first_net_module;
+  }
```

Comment:
This seems to cover only the trivial single module case.  Do you plan to write finding the common ancestor?